apps/Makefile

```diff
@@ -25,6 +43,19 @@ deploy-app-%:
 	. ../config/environment_$(deployment-stage) && helm upgrade -f $(deployment-stage).yaml $(*) $(*)  --set-string environment=$(deployment-stage),fastqImage=$(fastq-validator-image),image=$(image),replicas=$($(*)-replicas),gitlab_app=$(*),gitlab_env=$(deployment-stage) --wait --install
 	rm *.tgz
 
+deploy-all:
+	$(MAKE) deploy-secrets
+	$(MAKE) deploy-ontology
+	$(MAKE) deploy-app-ingest-state-tracking image=${INGEST_STATE_TRACKING_INITIAL_IMAGE}
```

Comment:
assuming the `INGEST_ROOT_API` [here](https://github.com/ebi-ait/ingest-state-tracking/blob/master/Dockerfile) isn't used in which case the order of core starting here is fine.

Reply:
I think this order is the appropriate order but @ke4 to confirm when he builds a cluster